.ci/docker/aotriton_version.txt

```diff
@@ -1,5 +1,5 @@
-0.6b
+0.7b
 manylinux_2_17
 rocm6.2
```

Comment:
CI is currently on rocm6.2, but I assume compatibility of this tar with rocm 6.1 also.

Reply:
According to the CI name I think they were on ROCm 6.1 and it seems there isn't compatibility problem yet.

I may file a separate PR later to let cmake pick the rocm version according to env var.

Reply:
That'd be a good change